src/CMakeLists.txt

```
@@ -24,8 +24,12 @@ add_library(MCControl SHARED ${MCControl_SRC} ${MCControl_HDR})
 add_dependencies(MCControl generate_mc_idl_cpp)
 set_target_properties(MCControl PROPERTIES PREFIX "")
 pkg_config_use_dependency(MCControl hrpsys-base)
-target_link_libraries(MCControl PUBLIC mc_rtc::mc_control)
-target_include_directories(MCControl PRIVATE "${PROJECT_BINARY_DIR}/idl")
+target_link_libraries(MCControl PUBLIC mc_rtc::mc_control "${HRPSYS_BASE_PREFIX}/lib/libhrpIo.so")
+target_include_directories(MCControl PRIVATE "${PROJECT_BINARY_DIR}/idl" "${HRPSYS_BASE_PREFIX}/include/hrpsys/io/")
```

Cleaner CMake way to do this:

```suggestion
if(NOT TARGET hrpIo::hrpIo)
  find_path(hrpIo_INCLUDE_DIR NAMES iob.h HINTS ${HRPSYS_BASE_PREFIX}/include/hrpsys/io)
  find_path(hrpIo_LIBRARY NAMES hrpIo HINTS ${HRPSYS_BASE_PREFIX})
  include(FindPackageHandleStandardArgs)
  find_package_handle_standard_args(hrpIo DEFAULT_MSG REQUIRED hrpIo_LIBRARY hrpIo_INCLUDE_DIR)
  mark_as_advanced(hrpIo_INCLUDE_DIR hrpIo_LIBRARY)
  add_library(hrpIo::hrpIo INTERFACE IMPORTED GLOBAL)
  set_target_properties(hrpIo::hrpIo PROPERTIES
    INTERFACE_INCLUDE_DIRECTORIES ${hrpIo_INCLUDE_DIR}
    INTERFACE_LINK_LIBRARIES ${hrpIo_LIBRARY}
  )
endif()
target_link_libraries(MCControl PUBLIC mc_rtc::mc_control hrpIo)
target_include_directories(MCControl PRIVATE "${PROJECT_BINARY_DIR}/idl")
```